docs/source/release/v6.1.0/sans.rst

```diff
@@ -9,11 +9,11 @@ SANS Changes
     putting new features at the top of the section, followed by
     improvements, followed by bug fixes.
 
-
-Bugs fixes
-----------
+Bugfixes
+--------
 
 - Fix a bug that made it impossible to process flux in SANSILLAutoprocess.
+- On D16 using :ref:`SANSILLAutoProcess <algm-SANSILLAutoProcess>`, now use the correct monitor for normalization, fix a bug regarding transmission processing at 90 degrees, and improve the q binning used.
```

Comment:
Just a real nit pick, could you clarify what "a bug" was. Such as "fix a bug causing x to happen when transmission processing at 90 degrees".

This will save Daniel having to ask when he's prepping the release notes for the upcoming release and allows users to clearly identify what should be resolved. Apart from that, I'm happy to merge this

Reply:
That would also trigger the builds again, which seem to be stuck.

Reply:
Ah, I've seen this before and I believe Sam is looking into it, let me trigger the build manually for this one

If you do a docs only change I'll merge is as soon as the doc tests (which are the fastest) are done to avoid it sitting in PR hell

Reply:
There, it should be clearer now hopefully. Thanks !